src/codecs.ts

```diff
@@ -871,3 +873,61 @@ export const CookieCsvInput = t.intersection([
 
 /** Type override */
 export type CookieCsvInput = t.TypeOf<typeof CookieCsvInput>;
+
+/// //////////////////////////////////////
+// Transcend Proxy AI Server policies   //
+/// //////////////////////////////////////
+export const Policies = t.literal('redactEmail');
+
+/** Type override */
+export type Policies = t.TypeOf<typeof Policies>;
+
+/** the codec of a route enabled in an AI integration */
+export type EnabledRouteC = t.TypeC<{
+  /** the name of the enabled route */
+  routeName: t.Mixed;
+  /** the enabled policies */
+  enabledPolicies: t.ArrayC<t.LiteralC<Policies>>;
+}>;
+
+/** the codec of an AI Integration */
+export type AIIntegrationC = t.TypeC<{
+  /** the routes enabled in the AI integration */
+  enabledRoutes: t.ArrayC<EnabledRouteC>;
+}>;
+
+/**
+ * The names of the OpenAI routes that we support setting policies for
+ * reference: https://platform.openai.com/docs/api-reference/introduction
+ */
+export const OpenAIRouteName = t.union([
+  t.literal('/v1/chat/completions'),
+  t.literal('/v1/images/generations'),
+  t.literal('/v1/images/edits'),
```

Comment:
So far we are really only supporting `/v1/chat/completions` with embeddings up next. Idk if you want to change that here or if these are just examples.

Reply:
They were supposed to be just examples. I will remove them

Reply:
done